File: WorkingHours.java

public class WorkingHours {
	private int openingHour; 
	private int closingHour; 
	
	public WorkingHours(int openingHour, int closingHour) {
		this.openingHour = openingHour; 
		this.closingHour = closingHour; 
	}
	
	public int getOpeningHour() {
		return openingHour; 
	}
	
	public int getClosingHour() {
		return closingHour; 
	}
	
	public String toString() {
		return "Opening hour: " + openingHour + ". Closing Hour: " + closingHour; 
	}
}

File: WorkingHours.cs
using System;
namespace Library
{
    public class WorkingHours
    {
        /* This class represents a set of operating hours. 
         * It has an opening hours and a closing hour.
         */

        private int openingHour;
        private int closingHour; 

        // constructor -- input the opening and closing hour 
        public WorkingHours(int openingHour, int closingHour)
        {
            this.openingHour = openingHour;
            this.closingHour = closingHour; 
        }

    }
}
